modules/single_page_checkout/reg_steps/payment_options/EE_SPCO_Reg_Step_Payment_Options.class.php

```diff
@@ -2410,17 +2410,21 @@ private function _post_payment_processing($payment = null)
                 $payment->set_status(EEM_Payment::status_id_pending);
                 $payment->save();
             } else {
-                // not a payment
+                // we couldn't redirect the user. Let's tell them why.
+                $error_message = sprintf(
+                    esc_html__(
+                        'It appears the Off Site Payment Method was not configured properly.%sPlease try again or contact %s for assistance.',
```

Comment:
We should always use numbered placeholders (`%1$s`, `%2$s`, etc) when there are more than one just in case some other language needs to rearrange the order of things.
I know it seems unlikely in this case, given the sentence, but you never know, and it's a good habit we should follow.

Reply:
Thanks for the reminder. Although I rearranged the code, that exact string existed previously, and I'm pretty sure changing it to numbered placeholders would break the existing translations. Assuming that's right, I think it's best to leave that string as it was before this PR.
But ya, I'm on board that new strings should always use the numbered placeholders as a habit.